net.sourceforge.Artha.yaml

```diff
@@ -0,0 +1,140 @@
+app-id: net.sourceforge.Artha
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: artha
+
+rename-desktop-file: artha.desktop
+#rename-icon: artha.png
+
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  # dbus session bus for artha
+  - --own-name=org.artha.unique
+  
+modules:
+
+- shared-modules/gtk2/gtk2.json 
+    
+# Wordnet 3.0 Needs tcl 8.5.x 
+- name: tcl8.5.19
+  cleanup: 
```

Comment:
Instead of duplicating the cleanups make it top level. This makes the manifest long.